lib/pbench/test/unit/server/query_apis/test_controllers_list.py

```diff
@@ -25,11 +25,27 @@ def build_index(self, server_config, dates):
             index += f"{idx}{d},"
         return index
 
-    def test_missing_json_object(self, client, server_config):
+    def test_non_accessible_user_data(self, client, server_config, pbench_token):
+        """
+        test_missing_json_object Test behavior when Authorization header does not have access to other user's data
+        """
+        # Auth token belongs to the user "drb" because the pytest fixture pbench_token creates it for the user "drb".
```

Comment:
That's a bit long and convoluted. How about "The pbench_token fixture logs in as user "drb""